WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/download/usecases/GetBackupDownloadStatusUseCase.kt

```diff
@@ -31,6 +33,8 @@ class GetBackupDownloadStatusUseCase @Inject constructor(
             val result = activityLogStore.fetchBackupDownloadState(FetchBackupDownloadStatePayload(site))
             if (result.isError) {
                 if (retryAttempts++ >= MAX_RETRY) {
+                    AppLog.e(T.JETPACK_BACKUP_DOWNLOAD,
+                            "GetBackupDownloadStatusUseCase: Exceeded 3 retries while fetching status")
```

Comment:
Minor (🔍): Consider using `${this.javaClass.simpleName}` instead of hardcoded class name, just like you did with the other two app logs.

Reply:
> Minor (🔍): Consider using `${this.javaClass.simpleName}` instead of hardcoded class name, just like you did with the other two app logs.

That was explicit because when in coroutine you don't get the name of the outer class .. you get an obscure reference to "coroutine" or in this case the message would have been "SafeCollector: Exceeded 3 retries while fetching status". I want the actual name of the class that is throwing the error.

Thanks for taking a quick look. Still waiting on the Utils piece to be sorted. :)

Reply:
Hi @ParaskP7 

The utils pieces have been merged in. Once CI finishes we should be good to go. Thanks again!

Reply:
👋 @zwarm !

Sorry, I was too quick into the `{this.javaClass.simpleName} and thanks for noting that mistake. What about this alternative: `this@GetBackupDownloadStatusUseCase.javaClass.simpleName`

Reply:
PS: I am seeing the CI complaining still, is that because we need to trigger another run?

Reply:
> PS: I am seeing the CI complaining still, is that because we need to trigger another run?

Me too ... 👀  as I'm not sure why the latest doesn't get pulled in. It's not complaining about the pom like it does for FluxC.

Reply:
Maybe @malinajirka can help? Shameless mention... 😄

Reply:
>  `this@GetBackupDownloadStatusUseCase.javaClass.simpleName`

I bet this _would_ work .. let me test. 👍

Reply:
It seems `1.30.3-beta.1` doesn't exist. 

I think the build is not failing on missing pom because the Utils library is referenced (transitive dependency) from other dependencies we rely on. So it fetches a different version of the utils library and that's enough for gradle - however, the other version is missing the newest changes, hence the build fails. Wdyt?

Reply:
Another comment - we also have a duplicate version of AppLog in `org.wordpress.android.util`. Perhaps we need to add `JETPACK_BACKUP` there as well. Wdyt?

Reply:
> Another comment - we also have a duplicate version of AppLog in test/org.wordpress.android.util. Perhaps we need to add JETPACK_BACKUP there as well. Wdyt?

Yes, that would explain the failing test with java.lang.NoSuchFieldError. 🙇

Reply:
@ParaskP7 - finally. Many thank to Jirka, that failing test was driving me nuts - that's a great little nugget of WP knowledge to stash away.